### PR TITLE
Re-organization of content

### PR DESCRIPTION
**Pull request by alexandrascript**
Our documentation can be better organized, creating a clearer journey for our readers to work with Perspective API. This PR will re-organize existing content and also takes relevant content from [the Perspective API support site](https://support.perspectiveapi.com/s/) to clarify more of our documentation.

### CODE REVIEW

File: 1-get-started/about.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[Perspective API documentation](../README.md) > [Get Started](README.md) > **About the API**
+
+# About the API
+
+Perspective is an API that makes it easier to host better conversations. The API uses machine learning models to score the perceived impact a comment might have on a conversation. Developers and publishers can use this score to give realtime feedback to commenters or help moderators do their job, or allow readers to more easily find relevant information.
+
+Start using the API by creating a new Google Cloud project, generating an API key, and making your first request. Build projects using production models or explore using experimental models.
+
+## Create tools for readers
```
Comment (oliviaredfield) [inline 1-get-started/about.md:14]: WDYT about linking to open source code for reader and moderator sections here?

File: 2-api/key-concepts.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Key concepts**
```
Comment (oliviaredfield) [inline 2-api/key-concepts.md:1]: Is this the same as the glossary? Should we merge those two docs if so?

File: 2-api/limits.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+Visit the Perspective API Support page to [request a quota increase](https://support.perspectiveapi.com/s/request-quota-increase).
+
+## Character limit for requests
+
+The maximum text size per request is 3000 bytes.
+
+## Error messages
+
+When using Outline, you might see a variety of error messages. This article details what each of those error messages might mean.
```
Comment (oliviaredfield) [inline 2-api/limits.md]: "Outline" is a separate product - did this info appear in the Support site for Perspective?

File: 2-api/limits.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **LImits**
```
Comment (oliviaredfield) [inline 2-api/limits.md]: LImits -> Limits

File: 2-api/methods.md [hunk 5]
```diff
@@ -0,0 +1,274 @@
+[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Methods**
+
+# API Methods
+
+## Scoring comments: `AnalyzeComment`
```
Comment (oliviaredfield) [inline 2-api/methods.md:10]: WDYT about breaking out AnalyzeComment and SuggestCommentScore into their own subpages in the Methods section?

File: 2-api/language.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **Languages**
+
+# Languages served by Perspective API
+
+## Languages in production
```
Comment (oliviaredfield) [inline 2-api/language.md]: My only hesitation with giving Language a separate doc is that we have to remember to update here and also in Model Reference when we have new launches. Maybe we could kill this page and rename Models to "Attributes and Languages" to better call out that language info is in there? Open to discussing!

**Review by oliviaredfield — COMMENTED** [review]
Left some comments - let's discuss when we meet!

File: 2-api/limits.md (hunk 3 above)
Comment (alexandrascript) [reply 2-api/limits.md]: It did indeed, so it definitely doesn't make sense.

File: 2-api/methods.md (hunk 5 above)
Comment (alexandrascript) [reply 2-api/methods.md:10]: Resolved by adding jump links at the top instead of separating

File: 2-api/language.md (hunk 6 above)
Comment (alexandrascript) [reply 2-api/language.md]: This was merged with the models page

**Review by alexandrascript — COMMENTED** [review]
All of the comments have been addressed! This is ready to merge, pending final approval.

**Pull request merged**